governance.md

```diff
@@ -106,15 +107,28 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
-To propose a new working group, first find a SIG to sponsor the group. 
+Working groups do not own any code or subprojects. Instead, they are a place for
+people to discuss topics that cross SIG boundaries.
+
+Working groups are primarily used to facilitate topics of discussion that are in
+scope for Kubernetes but that cross SIG lines. If a set of folks in the
+community want to get together and discuss a topic, they can do so without
+forming a Working Group. As a community we will be looking for other ways to
+highlight and encourage a larger ecosystem (with things like slack channels)
+without offering any official endorsement.
+
+To propose a new working group, first find a SIG to sponsor the group.
```

Comment:
Given that WG's are intended to cross SIGs, I think that multiple sigs should sponsor.  If only one SIG is in support of the WG, it can simply live in the SIG with all other discussions and/or sub-projects there.